app/views/bulk_actions/new.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class='container'>
   <h1>New Bulk Action</h1>
 
-  <%= form_for(@bulk_action, html: { 'data-controller' => 'bulk_actions' }) do |f| %>
-    <% if @bulk_action.errors.any? %>
+  <%= form_for(@form, html: { 'data-controller' => 'bulk_actions' }) do |f| %>
+    <% if @form.errors.any? %>
       <div id="error_explanation">
-        <h2><%= pluralize(@bulk_action.errors.count, 'error') %> prohibited this bulk_action from being saved:</h2>
+        <h2><%= pluralize(@form.errors.count, 'error') %> prohibited this bulk_action from being saved:</h2>
 
         <ul>
-        <% @bulk_action.errors.full_messages.each do |message| %>
+        <% @form.errors.each do |message| %>
```

Comment:
Are the "full" messages still yielded here?

Reply:
I've restored this.